src/lupus-slider.vue

```diff
@@ -102,6 +155,12 @@
       this.transformSlide(this.$refs.lupusSlider.swiper.activeIndex + 1);
     },
     methods: {
+      slideTo(slide, speed, callback) {
```

Comment:
`callback` is not used

Reply:
I know but it's nice to have as a feature, being able to perform something after slide change

Reply:
I agree, but it won't be called, I don't see it used anywhere?